.circleci/config.yml

```diff
@@ -15,6 +15,15 @@ only-on-release: &only-on-release
       ignore: /.*/
 
 jobs:
+  build-docs:
+    working_directory: ~/marquez
+    docker:
+      - image: circleci/node:12.21.0
```

Comment:
`cimg/node:12.21` might be better: https://circleci.com/blog/announcing-our-next-generation-convenience-images-smaller-faster-more-deterministic/

and omitting patch version will always pull the latest one among minor versions.

Reply:
Ahh, good point 👍